packages/react-components/react-alert/src/stories/Alert/AlertAvatar.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import { DismissCircleRegular } from '@fluentui/react-icons';
+
+import { Alert } from '../../index';
+
+export const AvatarProp = () => (
+  <div style={{ display: 'flex', flexDirection: 'column', gap: '10px' }}>
+    <Alert
+      action={{
+        appearance: 'transparent',
+        children: 'Undo',
+      }}
+      avatar={{ name: 'Jane Doe', size: 24, color: 'dark-red' }}
+    >
+      sent you a message
+    </Alert>
+    <Alert
+      action={{
+        appearance: 'transparent',
```

Comment:
```suggestion
```

I overlooked this in the previous PR, but let's move `appearance` to `defaultProps` of the slot so users will not need to specify it by default:

```diff
-action: resolveShorthand(props.action),
+action: resolveShorthand(props.action, {
+  defaultProps: { appearance: 'transparent' },
+}),
```

Please also update other `Alert` stories.

Reply:
Great suggestion, I didn't know this was possible. Thank you!